samples/androidNativeActivity/src/main/kotlin/renderer.kt

```diff
@@ -306,10 +306,10 @@ class Renderer(val parentArena: NativePlacement, val nativeActivity: ANativeActi
         }
 
         glFrontFace(GL_CW)
-        glVertexPointer(4, GL_FLOAT, 0, vertices.toFloatArray().toCValues().getPointer(this))
-        glTexCoordPointer(2, GL_FLOAT, 0, texCoords.toFloatArray().toCValues().getPointer(this))
-        glNormalPointer(GL_FLOAT, 0, normals.toFloatArray().toCValues().getPointer(this))
-        glDrawElements(GL_TRIANGLES, triangles.size, GL_UNSIGNED_BYTE, triangles.toByteArray().toCValues().getPointer(this))
+        glVertexPointer(4, GL_FLOAT, 0, vertices.toFloatArray().toCValues().ptr)
```

Comment:
Looks like `toCValues().ptr` is a rather common operation. Do you think we could add a shortcut for it?

Reply:
Problem is that we have multiple toCValues() methods, so we must provides multiple shortcuts then, and semantics would be pretty magical.